### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by davidlehn**
- The tests don't work with `@digitalbazaar/did-method-key@5`.

### CODE REVIEW

File: tests/20-main.spec.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
 /*!
  * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
```
Comment (dlongley) [inline tests/20-main.spec.js:2]:
```suggestion
 * Copyright (c) 2022-2024 Digital Bazaar, Inc. All rights reserved.
```

File: lib/index.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 /*!
  * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
```
Comment (dlongley) [inline lib/index.js:2]:
```suggestion
 * Copyright (c) 2022-2024 Digital Bazaar, Inc. All rights reserved.
```

**Review by dlongley — APPROVED** [review]
No worries about the older did-method-key lib, that can be upgraded in the test suite later.

**Review by aljones15 — APPROVED** [review]
Looks good and thanks.

**Pull request merged**